src/assets/wise5/vle/student-account-menu/student-account-menu.component.ts

```diff
@@ -43,11 +45,26 @@ export class StudentAccountMenuComponent implements OnInit {
     this.rootNodeStatus = this.nodeStatuses[this.rootNode.id];
     this.workgroupId = this.configService.getWorkgroupId();
     this.usersInWorkgroup = this.configService.getUsernamesByWorkgroupId(this.workgroupId);
+    this.usernamesDisplay = this.getUsernamesDisplay(this.usersInWorkgroup);
+    this.updateScores();
+    if (this.maxScore != null) {
+      this.subscriptions.add(
+        this.studentDataService.nodeStatusesChanged$.subscribe(() => {
+          this.updateScores();
+        })
+      );
+    }
```

Comment:
What is the maxScore null check for? Don't we want the scores to update even if maxScore is null?

Reply:
We only show scores in the template if maxScore is not null. I refactored based on our discussion.